pkg/env/node_scan.go

```diff
@@ -23,4 +23,7 @@ var (
 
 	// NodeScanningMaxBackoff is the upper boundary of backoff. Defaults to 5m in seconds, being 50% of Kubernetes restart policy stability timer.
 	NodeScanningMaxBackoff = registerDurationSetting("ROX_NODE_SCANNING_MAX_BACKOFF", 300*time.Second)
+
+	// NodeAnalysisDeadline is a time in which node analyzer should finish filesystem scanning and produce an inventory
+	NodeAnalysisDeadline = registerDurationSetting("ROX_NODE_SCAN_DEADLINE", 15*time.Second)
```

Comment:
I would be in favour of unifying the env var names. For the caching & backoff I also went with node scanning as introduced by Victor.
I'm open to using node scan as well, but then I'd recommend a rebase on master and changing all env vars including caching & backoff at the same time
```suggestion
	NodeScanningDeadline = registerDurationSetting("ROX_NODE_SCANNING_DEADLINE", 15*time.Second)
```